### ISSUE TITLE
[Request]: Release CC-app

### ISSUE THREAD

**Issue by lukey-luke** [department-of-veterans-affairs/abd-vro#2607]
### Requesting Team

Contention Classification

### Related product/initiative

Monitoring

### Problem
Additional logging statements have been made in order to more accurately determine # of NEW claims vs Claims for Increase.
`claim_type` is used for new widgets on datadog dashboard [HERE](https://lighthousedi.ddog-gov.com/dashboard/dy3-ybp-gdd/vro-contention-classification-metrics?refresh_mode=sliding&view=spans&from_ts=1705247431270&to_ts=1707839431270&live=true).

### Need
The `develop` branch has new commits that are ready-for-release to the cc-app container.
I am requesting that VRO release the `develop` branch to production for cc-app.

### Why?

This will allow us to accurately communicate metrics to stakeholders.

### Acceptance criteria

New release for `develop` branch of abd-vro is released

### Priority

medium?

**Comment by lukey-luke** [comment]
Any updates on this release? What's an ETA on releasing these changes?

**Comment by nelsestu** [comment]
@lukey-luke I am not sure why the last deploy didn't seem to have an effect, but I've redeployed to both QA and Dev and have validated that the domain-cc pods have the image tag that I am expecting.  Please have another look, and feel free to hit me up directly if there is still something amiss.

**Comment by ghost** [comment]
@nelsestu hey there! We are not seeing these changes reflected in the production logs yet. Could you please advise? Thanks!

cc @meganhicks @lukey-luke @tyler-spangler6 @yurenanava

**Comment by nelsestu** [comment]
> @nelsestu hey there! We are not seeing these changes reflected in the production logs yet. Could you please advise? Thanks!
> 
Let me first apologize for my confusion, and also acknowledge that I am finally understanding what that confusion is.  Is it typical to skip over a staged rollout?  By that I mean that we have five LHDI environments: dev, qa, sandbox, prod-test and prod.  While it seems normal to bypass some of the intermediate release stages, I've never encountered a context where all intermediate stages are bypassed.  I took it for granted that releasing the develop branch to production implies a staged rollout.  In this case I deployed to dev and qa, and notified about that deployment. Then the CC team would validate and approve that deployment.  Your approval would then be my prompt to proceed and so on and so forth. 

Of course in hindsight I feel a little ridiculous that the literal meaning of your request never even crossed my mind, but that is the truth about why we continue to miss each other.  In the interest of avoiding this confusion in the future, I ask... is this typical?  Once we get this cleared up, I promise not to be so thick in the future.

**Comment by lukey-luke** [comment]
I think that there might be some confusion are `DEV` and `develop`. AFAIK, the `develop` branch of abd-vro is the main, stable branch and should be production-ready at all times.

I previously requested a deployment to a `DEV` environment for some more experimental changes. See associated slack thread [here](https://dsva.slack.com/archives/C01Q7979Z7D/p1707429386298359).

I think we should assume code in `develop` is ready for production. Please let me know if I missed anything!! :)

**Comment by lukey-luke** [comment]
cc @bianca-rivera

**Comment by nelsestu** [comment]
@lukey-luke I can understand why you might think I am confused about the difference between the LHDI environment named Dev and the Git default branch named develop.  There are plenty of times when that confusion can play a roll in misunderstandings, but I am not confusing the two in this case. But please allow me to back up again, and I'll again try to restart once again by apologizing for the misunderstanding. I really am sorry for the misunderstanding.  This was my mistake, and I will do what I can to correct it, so let's please try to proceed with the clarity that 
a) We have ongoing efforts to improve the request process
b) it is true that we prefer to assume develop contains production ready code
c) it is also true that we will be standardizing on the staged deployment process that I referenced previously
d) we will work with the CC team to identify a process that works well for both teams

To clarify the current state of production deployments, we are presently blocked due to SecRel vulnerabilities that were flagged on Monday.  We have multiple team members working to unblock deployments already and having reopenned this ticket, I will circle back to this deployment just as soon as we are able to deploy.
 
We'll continue to work towards creating a clear deployment request process.  Knowing that you've already done that step is new information to me, and certainly relevant to my previous question about staged deployments.   I am pushing towards standardizing on branch name conventions which will include tagged release versions being the only images that are bound for production.  Lastly, please have a look at [this upcoming work item in our backlog](https://github.com/department-of-veterans-affairs/abd-vro/issues/2464). Together I hope to land on a process that works better for everyone and results in less confusion in the future and thanks for your patience.

**Comment by nelsestu** [comment]
At long last, Mar 6, 2024, 10:40 AM MST, the domain-cc pod has been updated in production which is now running domain-cc image tag: [ac8b5d4](https://ghcr.io/department-of-veterans-affairs/abd-vro-internal/vro-cc-app:ac8b5d4). Previous to [this release](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/8176376014) production domain-cc was running image [0e5e9e9](https://ghcr.io/department-of-veterans-affairs/abd-vro-internal/vro-cc-app:0e5e9e9) 
We were finally able to get clear of the SecRel issues blocking production deployments on Tuesday 3/5. Once unblocked Mason started walking through each service in each environment redeploying where necessary to address
